File: workspace/data-model/src/main/java/de/pfiva/data/model/snips/Range.java
package de.pfiva.data.model.snips;

import java.io.Serializable;

public class Range implements Serializable {

	private static final long serialVersionUID = 1L;
	
	private int start;
	private int end;
	
	public int getStart() {
		return start;
	}
	public void setStart(int start) {
		this.start = start;
	}
	public int getEnd() {
		return end;
	}
	public void setEnd(int end) {
		this.end = end;
	}
	@Override
	public String toString() {
		return "Range [start=" + start + ", end=" + end + "]";
	}
}

File: angular_workspace/pfiva-web/src/app/data-model/snips/Range.ts
export class Range {
    private _start: number;
    private _end: number;
    
	public get start(): number {
		return this._start;
	}

	public set start(value: number) {
		this._start = value;
	}

	public get end(): number {
		return this._end;
	}

	public set end(value: number) {
		this._end = value;
	}    
}
